test/org/loklak/tools/storage/AsciiTest.java

```diff
@@ -0,0 +1,54 @@
+package org.loklak.tools.storage;
+
+import org.loklak.tools.ASCII;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+/**
+ * These unit-tests test org.loklak.tools.ASCII.java
+ */
+public class AsciiTest {
+    /**
+    * This method tests 'compare' method of org.loklak.tools.ASCII.java
+    */
+    @Test
+    public void testCompare() {
+
+        ASCII ascii = new ASCII(true);
+
+        String s0 = "abc";
+        String s1 = "xyz";
+        int returnedValue1 = ascii.compare(s0, s1);
+        int returnedValue2 = ascii.compare(s1, s0);
+        int returnedValue3 = ascii.compare(null, null);
+        int returnedValue4 = ascii.compare(s0, null);
+        int returnedValue5 = ascii.compare(null, s1);
+        assertEquals(returnedValue1, -1);
```

Comment:
The first argument is `expected`.

https://stackoverflow.com/questions/26102865/assertequals-what-is-actual-and-what-is-expected/26104539